### PR TITLE
Add support for Visual Studio 2019, dropping support for versions <14.0

### PR DESCRIPTION
**Pull request by iam-mholle**
Addressing issue #103

### CODE REVIEW

File: LicenseHeaderManager/LicenseHeaderManager.csproj [hunk 1]
```diff
@@ -79,7 +81,8 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
-    <DeployExtension>True</DeployExtension>
+    <DeployExtension>False</DeployExtension>
```
Comment (FlorianDecker) [inline LicenseHeaderManager/LicenseHeaderManager.csproj]:
```suggestion
    <DeployExtension>True</DeployExtension>
```

Unnecessary change

File: LicenseHeaderManager/VsixInformation/source.extension.vsixmanifest [hunk 2]
```diff
@@ -11,17 +11,15 @@
...
+        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[14.0, 16.0]" />
+        <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[14.0, 16.0]" />
+        <InstallationTarget Id="Microsoft.VisualStudio.Enterprise" Version="[14.0, 16.0]" />
     </Installation>
     <Dependencies>
         <Dependency Id="Microsoft.Framework.NDP" DisplayName="Microsoft .NET Framework" d:Source="Manual" Version="[4.0,)" />
     </Dependencies>
     <Prerequisites>
-        <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.0,16.0)" DisplayName="Visual Studio core editor" />
+        <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.0,16.0]" DisplayName="Visual Studio core editor" />
```
[the remark this answers is not in the document]
Comment (iam-mholle) [reply LicenseHeaderManager/VsixInformation/source.extension.vsixmanifest]: Fixed in https://github.com/rubicon-oss/LicenseHeaderManager/pull/104/commits/a85eff4c1069a2938866ac5c099c31771ea1e651

File: LicenseHeaderManager/LicenseHeaderManager.csproj (hunk 1 above)
Comment (iam-mholle) [reply LicenseHeaderManager/LicenseHeaderManager.csproj]: Fixed in https://github.com/rubicon-oss/LicenseHeaderManager/pull/104/commits/3376d23e852fc8036c284627aa3a2145e9c618c5

**Comment by Stefan-Ilic** [comment]
Merged manually
